codes/classical/q-ary_digits/quasi_cyclic.yml

```diff
@@ -10,8 +10,10 @@ logical: q-ary_digits
 name: 'Quasi-cyclic code'
```

Comment:
The code file is now in a different location: /codes/classical/properties/quasi_cyclic.yml